### PR TITLE
[Relay][Runtime] Add simplify inference pass in the Relay interpreter

### PR DESCRIPTION
**Pull request by icemelon9**
Add simplify inference pass in the Relay interpreter so that it can execute models with batch norm and dropout.

### CODE REVIEW

**Comment by jroesch** [comment]
👍

**Comment by tqchen** [comment]
I want to note that maybe we will eventually need to revert this in the long run, if we start to work on training, as such transformation is not valid for training

**Comment by icemelon9** [comment]
Agree. This is a temporary fix to enable interpreter for more models. We definitely need to add ops for randomization in future.
